### ISSUE TITLE
the tooltip footer row placed  wrong

### ISSUE THREAD

**Issue by ElsaOOo** [SoorajSNBlaze333/tooltip-sequence#5]
![image](https://user-images.githubusercontent.com/48074435/93574489-0f625000-f9cb-11ea-968a-dd3622981efb.png)

seems the tooltip footer row placed  wrong.
here is the code 

"../dist/index.js" is your package.
```html
<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>test tooltip sequence</title>
  <link rel="stylesheet"
    href="https://unpkg.com/tooltip-sequence@latest/dist/index.css">
  <style>
    #element {
      width: 200px;
      height: 200px;
      background-color: yellow;
    }
  </style>
</head>

<body>
  <div id="element">target element</div>
  <script src="../dist/index.js"></script>
  <script>
    const options = {
      backdropColor: "transparent || color-code",
      sequence: [{
        element: "#element",
        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
        placement: "bottom"
      }],
      onComplete: function () {
        // your code
      }
    };
    createSequence(options);
  </script>
</body>

</html>

```

**Comment by SoorajSNBlaze333** [comment]
Hey @ElsaOOo. Thank you for noticing this issue. The issue is happening because I **missed** to add the border-sizing property to the css file. This will be fixed in the next release. To bypass the mis-alignment you can add this to your style tag

```css
*, ::after, ::before {
   box-sizing: border-box;
}
```

**Comment by ElsaOOo** [comment]
@SoorajSNBlaze333 nice!

**Comment by SoorajSNBlaze333** [comment]
Fixed in release **1.0.4**
